src/backend/distributed/citus--6.1-12--6.1-13.sql

```diff
@@ -0,0 +1,8 @@
+/* citus--6.1-12--6.1-13.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE INDEX pg_dist_partition_partmethod_index
```

Comment:
WIth its current implementation, we get `ERROR:  cache lookup failed for pg_dist_partition_partmethod_index, called to early?`  before executing `ALTER EXTENSION`. I'll be looking into more